recipes/meshpy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "meshpy" %}
+{% set version = "2016.1" %}
+{% set sha256 = "a9e80c1f23f6a6cdc66f8f62ea63dc273664f357d25e572bc5edf5f9b22af7d9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt && cd test && python -m pytest
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytools
+    - numpy
+    - six
+  run:
+    - python
+    - setuptools
+    - pytools
+    - numpy
+    - six
+
+test:
+  requires:
+    - pytest
+  imports:
+    - meshpy
+
+about:
+  home: http://github.com/inducer/meshpy
+  license: MIT and Proprietary
```

Comment:
> Both of those are non-essential. What's left at that point is an interface to the `gmsh` mesh generator.

You are the expert here and if they are non-essential and you think that removing them will delivery a package that is OK go ahead.

> What do you suggest?

If you do choose to ship those than the non-comercial part must be clear and the license files must be shipped with the package. I see you are adding the `doc/misc.rst` is that all the license text?